core/src/test/scala/org/apache/spark/scheduler/EventLoggingListenerSuite.scala

```diff
@@ -599,7 +599,8 @@ class EventLoggingListenerSuite extends SparkFunSuite with LocalSparkContext wit
       stageId: Int,
       taskType: String,
       executorMetrics: ExecutorMetrics): SparkListenerTaskEnd = {
-    val taskInfo = new TaskInfo(taskId, taskIndex, 0, 1553291556000L, executorId, "executor",
+    val taskInfo = new TaskInfo(
+      taskId, taskIndex, 0, taskIndex, 1553291556000L, executorId, "executor",
```

Comment:
nit: We should rename `taskIndex` as `partitionId` in this method - taskIndex happens to be partition id here.

Reply:
change `taskId, taskIndex, 0, taskIndex` to `taskId, taskIndex, 0, partitionId = taskIndex` is sound good to me.